doc/operator-scope.md

```diff
@@ -31,12 +31,15 @@ The SDK scaffolds operators to be namespaced by default but with a few modificat
 
 * `deploy/operator.yaml`:
   * Set `WATCH_NAMESPACE=""` to watch all namespaces instead of setting it to the pod's namespace
+  * Set `metadata.namespace` to define the namespace where the operator will be deployed.
 * `deploy/role.yaml`:
   * Use `ClusterRole` instead of `Role`
 * `deploy/role_binding.yaml`:
   * Use `ClusterRoleBinding` instead of `RoleBinding`
   * Use `ClusterRole` instead of `Role` for `roleRef`
   * Set the subject namespace to the namespace in which the operator is deployed.
+* `deploy/service_account.yaml`:
+  * Set `metadata.namespace` to the namespace where the operator is deployed.
```

Comment:
It is not required.

Reply:
This pull reqeust is about cluster wide resource watching. In this case there's a `ClusterRole` instead of `Role`. This also means that `ClusterRoleBinding`'s `subject.namespace` needs to be explicitly defined and it needs to match the ServiceAccount's namespace. 

If ServiceAccount's namespace is not explicitly defined (but it's implicitly deployed to the current kube-context)  there's a good chance that  ClusterRoleBinding will have an invalid explicit reference.

The implicit `default` namespace is a common source of confusion when some of your resources need to have an explicit reference but some of them are .